### PR TITLE
feature(component): campaign banner

### PR DESCRIPTION
**Pull request by gepifanio**
Ticket: [#TS-321](https://jira.unimelb.edu.au/browse/TS-321)

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 7553b8b7c590c250997d6ef571fe30a6178dcf1e

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5cee13caeccee200088b4ee5

**Comment by gepifanio** [comment]
Direct link to the banner: https://deploy-preview-1129--pattern-lib-unimelb.netlify.com/?selectedKind=Page%20%2F%20Header%20%2F%20Campaign&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

File: components/page/header/PageHeaderCampaign.vue [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    },
+    link: {
+      type: String,
+      default: '',
+    },
+  },
+};
+</script>
+
+<style scoped>
```
Comment (TimMansell) [inline components/page/header/PageHeaderCampaign.vue]: We shouldn't use scoped here as randomly generated classes could change in the future requiring CMS team to update.

File: components/page/header/PageHeaderCampaign.vue [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+    @media (--bp-wide) {
+      font-size: 4.25rem;
+      letter-spacing: -0.0625rem;
+      line-height: 4.125rem;
+    }
+  }
+}
+
+.btn--cta {
```
Comment (TimMansell) [inline components/page/header/PageHeaderCampaign.vue]: This should be targeted using a component class.  ie, `campaign__btn`

File: components/page/header/PageHeaderCampaign.vue [hunk 3]
```diff
@@ -0,0 +1,101 @@
+<template>
+  <div
+    :style="`background-image: url(${img})`"
+    class="campaign">
+    <div class="campaign__container">
+      <h1
+        v-if="title"
+        class="campaign__title">{{ title }}</h1>
+      <h1
```
Comment (TimMansell) [inline components/page/header/PageHeaderCampaign.vue]: For SEO and a11y reasons this should just be one `<h1>` or was this a design requirement?

File: components/page/header/PageHeaderCampaign.vue [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    },
+  },
+};
+</script>
+
+<style scoped>
+@import "../../_vars.css";
+
+.campaign {
+  background-position: 65%;
```
Comment (TimMansell) [inline components/page/header/PageHeaderCampaign.vue]: This should be 50% as default.

File: components/page/header/PageHeaderCampaign.vue [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  background-size: cover;
+  min-height: 386px;
+
+  @media (--bp-x-tablet) {
```
Comment (TimMansell) [inline components/page/header/PageHeaderCampaign.vue]: --bp-desktop suits better imo.

File: components/page/header/PageHeaderCampaign.vue (hunk 4 above)
Comment (TimMansell) [inline components/page/header/PageHeaderCampaign.vue]: We should move this value to a prop.  65% may be ideal for sample image but the campaign image may require another value.

File: components/page/header/PageHeaderCampaign.vue (hunk 3 above)
Comment (jaredh-um) [reply components/page/header/PageHeaderCampaign.vue]: Not a design requirement

**Pull request merged**